bloom-housing/bloom
# feat: changing auth over to cookies

# Pull Request Template

## Issue Overview

This PR addresses #[issue](https://github.com/bloom-housing/bloom/issues/2250)


## Description
This changes us over from auth bearer tokens to auth via cookies

doing so required us to create a pseudo-proxy by way of nextjs's api. So now all requests to the backend stop by the proxy first so cookies can be properly attached 

## How Can This Be Tested/Reviewed?
The best way to test this is to use the stood up environments: 
https://bloom-partners-orm.netlify.app/sign-in
and
https://bloom-core-orm.netlify.app/

testers should verify that basically all backend endpoints work correctly. This would include:

testing the flow of creating -> editing -> viewing details for -> viewing in public site for listings

testing the flow of creating -> editing -> viewing details for -> logging in as a user on both the public and parter site

testing the flow of applying for a listing on both the partner and public site

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
